cmd/eval.go

```diff
@@ -88,3 +36,51 @@ func validateFile(file []byte) error {
 	var holder interface{}
 	return yaml.Unmarshal(file, &holder)
 }
+
+var (
+	replacements map[string]string
```

Comment:
nit: Is a global variable needed here? Can this be defined in the RunE inline?

Reply:
Did some logic to remove this global variable. Makes things feel cleaner, good call.